smoketest/loggers.py

```diff
@@ -208,6 +223,14 @@ def log_success(self, url, test, result, response, platform, follow_redirects):
         result_desc = result.description
         elapsed = response.elapsed
         hops = _calculate_hops(response, follow_redirects)
+        formatted_request_headers = []
```

Comment:
Is there any reason not to make a helper method for this? It looks like this logic is repeated a few times.